flask/app/routes.py

```diff
@@ -340,3 +341,191 @@ def get_enums():
         if issubclass(obj, Enum) and name != 'Enum':
             enums[name] = [e.value for e in getattr(models, name)]
     return jsonify(enums)
+
+
+# TODO: text/csv content type does not seem to be working, ie. request.files['file'] throws a KeyError
+# TODO: decorator for the try...catch.. statements? 
```

Comment:
I'll look at if I can write a decorator for the repeated pattern or just two functions if not (one for flush, one for commit)